src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/SubmissionComponent.scala

```diff
@@ -107,6 +107,14 @@ trait SubmissionComponent {
       }))
     }
 
+    def countByStatus(workspaceContext: SlickWorkspaceContext): ReadAction[Map[String, Int]] = {
+      filter(_.workspaceId === workspaceContext.workspaceId).groupBy(s => s.status).map { case (status, submissions) =>
+        (status, submissions.length)
```

Comment:
[SO suggests](http://stackoverflow.com/a/2925643/2941784) you might be able to do:

```
blah.map({ case (status, submissions) =>
    status -> submissions.length
})(collection.breakOut)
```

Not sure if this is true, but seems less build-an-intermediate-list-y.